server/src/main/java/org/apache/uniffle/server/storage/HadoopStorageManager.java

```diff
@@ -65,12 +68,48 @@ public class HadoopStorageManager extends SingleStorageManager {
   private Map<String, HadoopStorage> appIdToStorages = JavaUtils.newConcurrentMap();
   private Map<String, HadoopStorage> pathToStorages = JavaUtils.newConcurrentMap();
   private final boolean isStorageAuditLogEnabled;
+  private final BlockingQueue<AsynchronousDeleteEvent> quickNeedDeletePaths =
+      Queues.newLinkedBlockingQueue();
+  private Thread clearNeedDeleteLocalPathThread;
 
   HadoopStorageManager(ShuffleServerConf conf) {
     super(conf);
     hadoopConf = conf.getHadoopConf();
     shuffleServerId = conf.getString(ShuffleServerConf.SHUFFLE_SERVER_ID, "shuffleServerId");
     isStorageAuditLogEnabled = conf.getBoolean(ShuffleServerConf.SERVER_STORAGE_AUDIT_LOG_ENABLED);
+    Runnable clearNeedDeletePathTask =
```

Comment:
Why not intergrating this part async deletion into the underlying class like SingleStorageManager for localfile and hadoop storage type to share

Reply:
All comments are complete.

Reply:
Got it. Let me take a look again